core/app/[locale]/layout.tsx

```diff
@@ -107,8 +108,11 @@ export default async function RootLayout({ params, children }: Props) {
   const messages = await getMessages();
 
   return (
-    <html lang={locale}>
-      <body className={`${inter.variable} ${dm_serif_text.variable} ${roboto_mono.variable}`}>
+    <html
+      className={clsx(inter.variable, dm_serif_text.variable, roboto_mono.variable)}
```

Comment:
nit: do we really need `clsx` here? 🙄

Reply:
idk we just never string interpolate classes like that anywhere so I changed it to be consistent